src/Sampler/RuleBased/src/ComponentProvider/SamplerRuleBased.php

```diff
@@ -49,17 +50,17 @@ public function createPlugin(array $properties, Context $context): SamplerInterf
         );
     }
 
-    public function getConfig(ComponentProviderRegistry $registry): ArrayNodeDefinition
+    public function getConfig(ComponentProviderRegistry $registry, NodeBuilder $builder): ArrayNodeDefinition
     {
-        $node = new ArrayNodeDefinition('contrib_rule_based');
+        $node = $builder->arrayNode('php_rule_based');
```

Comment:
My thinking here is that this is less likely to conflict with a sampler from another SIG, given that they would be unlikely to accept the same config